src/dkg-threshold/lit-protocol/v6.ts

```diff
@@ -131,16 +148,41 @@ export class LitProtocol {
 		stringHash: string,
 		unifiedAccessControlConditions: NonNullable<UnifiedAccessControlConditions>
 	): Promise<string> {
-		// generate auth signature
-		const authSig = await LitProtocol.generateAuthSignature(this.cosmosAuthWallet);
+		// mint capacity credits
+		const { capacityTokenIdStr } = await this.contractClient.mintCapacityCreditsNFT({
```

Comment:
@Eengineer1 can I separate minting capacity credits in a separate function and accept only `sessionSigs` / `capacityDelegationAuthSig` as a parameter to allow token to be minted by a separate entity and delegated to the current wallet address

Reply:
Please, synchronise your work with the `capacity-mint` branch, which has all the changes necessary to instantiate the underlying wallet.

We'll review the end result, but before proceeding please, prioritise the modifications present within that branch.

Reply:
The question is valid, in which case, already accounted for in the aforementioned implementation branch.